releases/release-1.18/release-notes-draft.md

```diff
@@ -1,142 +1,229 @@
-# v1.18.0-rc.1
+## Changelog since v1.17.0
```

Comment:
Please remove that heading, it’s already included by the generator.

Reply:
done